### PR TITLE
Make owner a relationship on Meme

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

File: examples/memeolist.tables.sql [hunk 1]
```diff
@@ -12,9 +12,8 @@ CREATE TABLE profile (
 CREATE TABLE meme (
   id SERIAL NOT NULL PRIMARY KEY,
   photourl CHARACTER VARYING(500) NOT NULL,
-  owner CHARACTER VARYING(100) NOT NULL,
   likes NUMERIC NOT NULL,
-  ownerid SERIAL NOT NULL references profile(id)
```
Comment (darahayes) [inline examples/memeolist.tables.sql]: Really happy with this PR except this one thing. I strongly believe this field should still be `ownerid` or `ownerId`. (whereas at a higher level, the resolved object will be called `owner`)

File: examples/memeolist.query.examples.graphql [hunk 2]
```diff
@@ -1,10 +1,14 @@
-## MEMES section
 query allMemes {
```
Comment (wtrocki) [inline examples/memeolist.query.examples.graphql]: This file is no longer used. We could remove it.

File: examples/memeolist.tables.sql (hunk 1 above)
Comment (wtrocki) [reply examples/memeolist.tables.sql]: Resolvers are now overly simplified (`select * from meme`) so table and graphql names are connected.

**Comment by wtrocki** [comment]
Great improvement!

File: examples/memeolist.tables.sql (hunk 1 above)
Comment (darahayes) [reply examples/memeolist.tables.sql]: Okay I will concede that one 😄

File: examples/memeolist.query.examples.graphql (hunk 2 above)
Comment (darahayes) [reply examples/memeolist.query.examples.graphql]: Can we keep it just for now? I've had some issues with the playground around variables. It tends to be a little buggy sometimes so I'd like to hold onto this file just for a short time

Comment (wtrocki) [reply examples/memeolist.query.examples.graphql]: Yeah! That was the reason to keep it.
We need variable format as server is now source of truth and we get those queries back to mobile,
but having separate one with variables is also good, but brings more maintenance cost.

**Comment by wtrocki** [comment]
@ziccardi To use it see: https://github.com/aerogear/data-sync-server#postgres-1

**Pull request merged**